help/actions/_posts/2015-02-09-export.md

```diff
@@ -74,27 +74,35 @@ The same format as above but encoded for use as a rich text document (rtf) which
+| symbol                                                                                                                         	| initials             	| different                	| first author: lastname, f. m.other authors: f. m., lastname      	| **%i**    	| , et al.             	|  -Henneken, E. A., G. Muench, L. Holm Nielsen, S. Blanco-Cuaresma, & A. Accomazzi<br>-Kurtz, M. J. & A. Accomazzi                   	|  -Henneken, E. A., G. Muench, et al.<br>-Kurtz, M. J. & A. Accomazzi                         	|
+| text                                                                                                                           	| initials             	| same                     	| lastname, f. m.                                                  	| **%L**    	| , and xxx colleagues 	|  -Henneken, E. A., Muench, G., Holm Nielsen, L., Blanco-Cuaresma, S., and Accomazzi, A.<br>-Kurtz, M. J. and Accomazzi, A.          	|  -Henneken, E. A., Muench, G., and 4 colleagues<br>-Kurtz, M. J. and Accomazzi, A.           	|
+| symbol                                                                                                                         	| initials             	| same                     	| lastname, f. m.                                                  	| **%l**    	| , et al.             	|  -Henneken, E. A., Muench, G., Holm Nielsen, L., Blanco-Cuaresma, S., & Accomazzi, A.<br>-Kurtz, M. J., & Accomazzi, A.             	|  -Henneken, E. A., Muench, G., et al.<br>-Kurtz and Accomazzi                                	|
+| none                                                                                                                           	| initials             	| same                     	| lastname, f. m.                                                  	| **%N**    	| , and xxx colleagues 	|  -Henneken, E. A., Muench, G., Holm Nielsen, L., Blanco-Cuaresma, S., Accomazzi, A.<br>-Kurtz, M. J., Accomazzi, A.                 	|  -Henneken, E. A., Muench, G., and 4 colleagues<br>-Kurtz, M. J., Accomazzi, A.              	|
+| text                                                                                                                           	| initials             	| same                     	| lastname f.m.note no space after first initial                   	| **%e**    	| , and xxx colleagues 	|  -Henneken, E.A., Muench, G., Holm Nielsen, L., Blanco-Cuaresma, S., and Accomazzi, A.<br>-Kurtz, M.J. and Accomazzi, A.            	|  -Henneken, E.A., Muench, G., and 4 colleagues<br>-Kurtz, M.J. and Accomazzi, A.             	|
+
+Note that formats %e and %f used to be known as %za2 and %za3 respectively. %za1 was eliminated since it is equivalent to %1H.
```

Comment:
Was this in Classic? May want to say something to that effect like `used to be known in Classic as`

Reply:
yes, converted %za2 and %za3  to %e and %f to have one character specifiers. that is why I have listed them at the bottom, that if someone is looking for the zs see them at last, by format, and not get lost between the other formats.

Reply:
Ok, that works